ooniprobe/View/TabsViewController.m

```diff
@@ -61,12 +63,17 @@ -(void)viewDidAppear:(BOOL)animated{
                           cancelButton:disableButton
                                 inView:self];
     }
+    //we don't want to flood the user with popups
+    else if ([SettingsUtility getAppOpenCount] == NOTIFICATION_POPUP_COUNT
+             && ![SettingsUtility isNotificationEnabled]){
+        [MessageUtility notificationAlertinView:self];
+        [SettingsUtility incrementAppOpenCount];
+    }
 }
 
 -(void)setModalValue:(BOOL)value key:(NSString*)key popupName:(NSString*)popupName{
     [[NSUserDefaults standardUserDefaults] setObject:@"ok" forKey:popupName];
     [[NSUserDefaults standardUserDefaults] setBool:value forKey:key];
-    [[NSUserDefaults standardUserDefaults] synchronize];
```

Comment:
this is deprecated and not needed anymore

Reply:
🙏